static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -445,20 +454,23 @@ $navigation-horizontal-padding: 38px;
 }
 
 .main-navigation__secondary {
-    background-color: darken($guardian-brand, 3%);
     color: #ffffff;
+    background-color: darken($guardian-brand, 3%);
     padding-bottom: $gs-baseline;
 }
 
 .navigation-border {
+    margin-top: -1px;
```

Comment:
is it the same as this line: https://github.com/guardian/frontend/pull/13972/files#diff-a6a8d02d22743e9a6dc47e407c359fb6L460. Is so, can we move the comment too?